include/translated-term.php

```diff
@@ -338,4 +338,81 @@ protected function get_db_infos() {
 			'default_alias' => 't',
 		);
 	}
+
+	/**
+	 * Wrapper of `wp_insert_term` with the language.
+	 *
+	 * @since 3.7
+	 *
+	 * @param string       $term     The term name to add.
+	 * @param string       $taxonomy The taxonomy to which to add the term.
+	 * @param PLL_Language $language The term language.
+	 * @param array|string $args     The taxonomy of the term.
+	 * @return array|WP_Error An array of the new term, WP_Error otherwise.
+	 *
+	 * @phpstan-param array{
+	 *   alias_of?: string,
+	 *   description?: string,
+	 *   parent?: int,
+	 *   slug?: string,
+	 * } $args
+	 */
+	public function insert_term( string $term, string $taxonomy, PLL_Language $language, $args = array() ) {
+		$set_language_for_term_slug = function () use ( $language ) {
+			return $language;
+		};
+
+		add_filter( 'pll_inserted_term_language', $set_language_for_term_slug, 20 ); // After Polylang's filter.
+		$term = wp_insert_term( $term, $taxonomy, $args );
+		remove_filter( 'pll_inserted_term_language', $set_language_for_term_slug, 20 );
+
+		if ( is_wp_error( $term ) ) {
+			// Something went wrong!
+			return $term;
+		}
+
+		$term_id = (int) $term['term_id'];
+		$this->model->term->set_language( $term_id, $language );
+
+		return $term;
+	}
+
+	/**
+	 * Wrapper of `wp_update_term` with the language.
+	 *
+	 * @since 3.7
+	 *
+	 * @param int          $term_id  The ID of the term.
+	 * @param string       $taxonomy The taxonomy of the term.
+	 * @param PLL_Language $language The term language.
+	 * @param array        $args     The taxonomy of the term.
```

Comment:
```suggestion
	 * @param array        $args     Array of arguments for updating a term.
```
According to WP doc.